### ISSUE TITLE
Queue only view

### ISSUE THREAD

**Issue by smalltimer** [quodlibet/quodlibet#3740]
The proposed change
-------------------
I propose a queue-only view where the browser and the filters are hidden. The client only shows the queue and the information for the currently playing song. Essentially, the browser and the filter are hidden. This could be implemented through a separate button.

Benefits of this change
-----------------------
This would enable users to create a queue, and then remove all clutter from the interface - showing only this queue. This would be especially helpful for users who set up a long queue (say, books, podcasts, multiple albums etc.) and once done with this, only want to have information to their queue. 

Any issues this might cause
---------------------------
I do not think this will cause any issues.

**Comment by declension** [comment]
Some (simpler?) ideas that might work:
1. a separate window (as a plugin, perhaps) that is not owned by its parent? So you'd launch the plugin but minimise the main QL
1. Make an extra queue "browser" that can be opened as normal as a secondary browser window
1. Just add a collapse control on the track list (like Queue now has)

**Comment by smalltimer** [comment]
I think 1. and 3. are great. I think 1. would be the easiest to implement, and perhaps also the cleanest. 3. seems like a lot of work but could open the doors to other GUI options like allowing users to choose exactly what they want to see (through buttons or preferences).

Also, does opening the issue/enhancement mean that I have to volunteer to do it. I'm afraid I do not have the skills or the free time to do so :( I'll of course close the issue right away if this is the case.

**Comment by declension** [comment]
> Also, does opening the issue/enhancement mean that I have to volunteer to do it. I'm afraid I do not have the skills or the free time to do so :( I'll of course close the issue right away if this is the case.

Not at all. Issues are open to everyone to post without too much expectation (from either side), but of course we're also happy for volunteers given the sizeable [issue list](./)

**Comment by eauchat** [comment]
I like the idea for there to be a "queue" browser.
It feels somehow simple enough to implement that I maybe could give a try to do it.
Would it be something that could be integrated if I make a PR for it?
Also I've never contributed to quodlibet code, so if there's anyone willing to list the files that would need to be modified for this to be implemented, that would be of great help :)
